src/data/primary-options.js

```diff
@@ -17,7 +17,8 @@ const options = [
   { value: 'debug', label: 'debug' },
   { value: 'recover', label: 'recover' },
   { value: 'synchronize', label: 'synchronize' },
-  { value: 'rebase', label: 'rebase' }
+  { value: 'rebase', label: 'rebase' },
+  { value: 'cherrypick', label: 'cherry-pick' },
```

Comment:
What do you think of adding this under the following steps to make it easier for newer users? 👇
- Dropdown 1: "Add"
- Dropdown 2: "A commit from another branch"

Reply:
It would be better.